### PR TITLE
Updates to Terraform provider

### PR DESCRIPTION
**Pull request by dehawkins512**
12932

### CODE REVIEW

**Comment by dehawkins512** [comment]
@miq-bot assign @Fryguy

File: managing_providers/_topics/configuration_management_providers.md [hunk 1]
```diff
@@ -15,9 +15,10 @@ Configuration management providers must be added individually to
...
+{{ site.data.product.title_short }} includes an IBM Terraform provider that provides functionality to integrate with the [Managed Services component](https://www.ibm.com/support/knowledgecenter/SSFC4F_2.2.0/cam/cam_intro.html). Managed services uses open source Terraform to manage and deliver cloud infrastructure as code. Terraform is an Open Source software that is developed by HashiCorp that enables predictable and consistent provisioning of many cloud platforms, classic infrastructure, and virtual private cloud (VPC) infrastructure resources by using a high-level scripting language. You can use Terraform to automate your cloud resource provisioning, rapidly build complex, multi-tier cloud environments, and enable Infrastructure as Code (IaC).
```
Comment (Fryguy) [inline managing_providers/_topics/configuration_management_providers.md:18]: The audience for this doc on upstream might not know the context of the CloudPak, so this line might need some clarification of Managed Services Component.  Thoughts?

```suggestion
{{ site.data.product.title_short }} includes an IBM Terraform provider that provides functionality to integrate with the [Managed Services component](https://www.ibm.com/support/knowledgecenter/SSFC4F_2.2.0/cam/cam_intro.html) of the IBM Cloud Pak for Multicloud Management. Managed services uses open source Terraform to manage and deliver cloud infrastructure as code. Terraform is an Open Source software that is developed by HashiCorp that enables predictable and consistent provisioning of many cloud platforms, classic infrastructure, and virtual private cloud (VPC) infrastructure resources by using a high-level scripting language. You can use Terraform to automate your cloud resource provisioning, rapidly build complex, multi-tier cloud environments, and enable Infrastructure as Code (IaC).
```

**Comment by chessbyte** [comment]
Ugh - Jason and I collided on this PR.  He commented at the same time as I merged.

**Comment by Fryguy** [comment]
It's ok...follow up PR is fine if you think it makes sense, @dehawkins512

File: managing_providers/_topics/triggering_a_refresh_of_a_terraform_provider.md [hunk 2]
```diff
@@ -6,4 +6,6 @@ Your IBM Terraform provider can still create new hosts independently of {{ site.
 
 2. Select your Terraform provider using the checkbox, and click menu: **Configuration > Refresh Relationships and Power States**. This triggers the refresh.
 
-3. When the refresh is complete, select the Terraform provider to check the updated list of hosts groups in the provider.
+3. When the refresh is complete, select the Terraform provider to check the updated list of Configuration profiles and Configured systems in the provider.
+
+
```
Comment (Fryguy) [inline managing_providers/_topics/triggering_a_refresh_of_a_terraform_provider.md:11]: Also noticed an unnecessary extra blank line here.

**Pull request merged**